bots/machine/machine_core/machine.py

```diff
@@ -51,6 +51,7 @@
 
 
 class Machine(ssh_connection.SSHConnection):
+    dhcp_server_pid_file = "/var/run/own_dnsmasq.pid"
     def __init__(self, address="127.0.0.1", image="unknown", verbose=False, label=None, browser=None,
```

Comment:
Please leave an empty line between the member variable and the method.

Also, please prefix the commit message with "test:".